book/contributions/rec_using_inchi/recipes/rec_using_inchi.md

```diff
@@ -0,0 +1,107 @@
+# Using InChI and InChIKey to annotate your chemical entities 
+
+```{dropdown} About this interactive ![icons](../static/img/rocket.png) recipe
+- Author: [Jordi Cuadros](https://orcid.org/0000-0001-6513-9140)
+- Reviewer:
+- Topics: InChI, InChIKey
+- Format: Tutorial
+- Scenarios: Retrieve the InChI and InChIKey of a chemical
+- Skills:
+    - Understand what an InChI is (<http://publications.iupac.org/ci/2006/2806/2806-pp12-15.pdf>, <https://www.inchi-trust.org/>)
+    - Know what MOL files are and where to get them (<https://en.wikipedia.org/wiki/Chemical_table_file>, <https://discover.3ds.com/sites/default/files/2020-08/biovia_ctfileformats_2020.pdf> )
+    - Understand what a SMILES string is (<https://en.wikipedia.org/wiki/Simplified_molecular-input_line-entry_system>, <http://opensmiles.org/>)
+- Learning outcomes: After completing this example you should understand:
+    - What makes InChI and InChIKey convenient identifier for chemical
+    - How to obtain the InChI and the InChIKey of a chemical
+    - How to identify a chemical behind an InChI or an InChIKey
+- Citation: 'Using InChI and InChIKey to annotate your chemical entities', The IUPAC FAIR Chemistry Cookbook, https://iupac.github.io/WFChemCookbook/recipes/rec_using_inchi.html
+- Reuse: This notebook is made available under a [CC-BY-4.0](https://creativecommons.org/licenses/by/4.0/) license.
+```
+
+## Scenario
+
+So far, we are quite used to identifying chemical substances with a name, a drawing or a CAS Registry Number (CAS-RN). However, these identifiers have some issues that affect their usability for indexing and searching. A way to increase the FAIRness of our documents is to annotate our chemicals with InChI and InChiKeys, either in papers, data, or metadata files.
+
+Let's do it together!
+
+## Case 1: I can draw my chemical 
+
+Many molecular drawing packages like ChemDraw, <https://revvitysignals.com/products/research/chemdraw>, ChemSketch, <https://www.acdlabs.com/products/chemsketch/>, or MarvinSketch, <https://chemaxon.com/marvin>, can output the InChI and the InChIKey for a molecule drawn on them.
+
```

Comment:
Could you add this link to a list of drawing packages? https://en.wikipedia.org/wiki/Molecule_editor